doc/api/dns.md

```diff
@@ -626,6 +626,17 @@ subsequent servers provided. Fallback DNS servers will only be used if the
 earlier ones time out or result in some other error.
 
 ## DNS promises API
+<!-- YAML
+added: v10.6.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/32953
+    description: Exposed as `require('dns/promises')`
```

Comment:
@Trott this change is in v15, and has not been released anywhere yet.

Reply:
Whoops! My mistake.